improver_tests/utilities/cube_manipulation/test_collapsed.py

```diff
@@ -76,6 +76,22 @@ def test_two_methods(self):
             ).all()
         )
 
+    def test_two_coords(self):
+        """Test behaviour collapsing over 2 coordinates, including not escalating
+        precision when collapsing a float coordinate (latitude)"""
+        result = collapsed(self.cube, ["realization", "latitude"], iris.analysis.MEAN)
+        self.assertTrue(
+            (
+                result.data
+                == self.cube.collapsed(
+                    ["realization", "latitude"], iris.analysis.MEAN
+                ).data
+            ).all()
+        )
```

Comment:
Black has left this looking a bit ugly. Could you use an `expected=` variable to make this a bit cleaner?